lib/membrane/video_compositor/output_options.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Membrane.VideoCompositor.OutputOptions do
+  @moduledoc """
+  Options of VideoCompositor output.
+  """
+
+  @typedoc """
+  After rendering VideoCompositors outputs are encoded.
+  Changing encoder preset can substantially impact output quality as well as CPU usage.
+  See [FFmpeg docs](https://trac.ffmpeg.org/wiki/Encode/H.264#Preset) to learn more.
+  """
+  @type encoder_preset ::
+          :ultrafast
+          | :superfast
+          | :veryfast
+          | :faster
+          | :fast
+          | :medium
+          | :slow
+          | :slower
+          | :veryslow
+          | :placebo
+
+  @enforce_keys [:width, :height, :id]
+  defstruct @enforce_keys ++ [encoder_preset: :fast]
```

Comment:
is the fast a default? I would prefer not to introduce any default values in a client SDK, if we want to have a default it should be in a compositor directlly